### PR TITLE
Separate cloudcore&admission in makefile

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

 /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,20 +6,24 @@ all:
 	cd edge && $(MAKE)
 	cd keadm && $(MAKE)
 	cd edgesite && $(MAKE)
-else ifeq ($(WHAT),cloud)
-# make all what=cloud, build cloud binary
+else ifeq ($(WHAT),cloudcore)
+# make all what=cloudcore, build cloudcore binary
```
Comment (kadisi) [inline Makefile]: # make all WHAT=cloudcore, build cloudcore binary

Comment (kadisi) [inline Makefile]: what -> WHAT

File: Makefile [hunk 2]
```diff
@@ -6,20 +6,24 @@ all:
...
 all:
-	cd cloud && $(MAKE)
-else ifeq ($(WHAT),edge)
+	cd cloud && $(MAKE) cloudcore
+else ifeq ($(WHAT),admission)
+# make all what=admission, build admission binary
```
Comment (kadisi) [inline Makefile]: what->WHAT

File: Makefile [hunk 3]
```diff
@@ -6,20 +6,24 @@ all:
...
+all:
+	cd cloud && $(MAKE) admission
+else ifeq ($(WHAT),edgecore)
 all:
-# make all what=edge, build edge binary
+# make all what=edgecore, build edgecore binary
```
Comment (kadisi) [inline Makefile]: what->WHAT

File: Makefile [hunk 4]
```diff
@@ -6,20 +6,24 @@ all:
...
 	cd edge && $(MAKE)
 else ifeq ($(WHAT),edgesite)
 all:
 	$(MAKE) -C edgesite
 else ifeq ($(WHAT),keadm)
 all:
-# make all what=edge, build edge binary
+# make all what=keadm, build keadm binary
```
Comment (kadisi) [inline Makefile]: what->WHAT

**Comment by sids-b** [comment]
Thanks @fisherxu 

/approve

**Pull request merged**